### PR TITLE
Update Standby Replication Diagrams

### PR DESCRIPTION
**Pull request by tjmoore4**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?
   - [ ] Have you added automated tests?

**Type of Changes:**

 
 - [ ] New feature
 - [ ] Bug fix
 - [x] Documentation
 - [ ] Testing enhancement
 - [ ] Other

**What is the new behavior (if this is a feature change)?**
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
This commit updates the existing repo-based standby cluster
configuration diagram to a new Draw.io generated image and adds
the associated XML file. It also creates two new diagrams to
illustrate a streaming standby cluster configuration and a cluster
that configured to have both a streaming standby and an external repo.

**Other Information**:
Issue: [sc-14710]

### CODE REVIEW

File: docs/static/drawio/repo-based-standby.xml [hunk 1]
```diff
@@ -0,0 +1 @@
...fDG6VlPRdXMWzwNrDz+CB5pZYOzpWujDDDzuFXIww650mY4C4QxQVzMV32JA9KanKCREFZ8MwckMr/GQLiZUzvGa8f9iJUWd9bJ2ef8ReJMHdUKCwxvWuL8qocQYPvY1Cu8wMnKfwnDYWXK9LkXiYlJ614uytjxUNiAQIJC5RFRLHEiwlWOGz5cJ1YSYF1gnc/0mn74BnEmIl6fteu7/IYpd/ymL/Vp0dMNLkNYmvdSb9iJETZuloB0tIu7dgLKhSCpGixLvQKHqm6EuizJ5Pma2S5iKoVK/83jEg8RwsKfIKaYdFCkoasfcaRYEuAdmx4uly9oCzgXVUe8x/Mokp2TR8VgHtJhXfq1L0EdcV8goqVmBbcOcoqlUDd5/gmFhYrYMsPa6BfxtF/3tddsF9rBLVRmHlYD5Jo7qcC3ts0IPgiod9ZBBzUq74if4VdS2ORMW50pL0Jep3DfctWLj3oHSvmJuknBrtzaX8YHoAzVBQ7Rr3uNG1fGu+BGKJjGpQvHeIctdyvW8CY5MSY+uS9RP693rPCWhCF3MAYkWmq4piONS+tQ8XXkToQ6gTwR47nOmkBTFuSKEWhqsadBl4R4iHlX971P+oLsX3wdWCv4O7Tmi3DI4AqiWZohQXVR1FTVBJPFEGHyqFgcM76RzcfQHZrzGpfI1sHkbekrirwv3t3WDFx5a5SoM7QdA/wt/AO0RF/D/NJzhPcImxA8b/zYGZRkv6R6P1uyMzqtL8h9b48zHK1j8a6i8Ozij/aP8fHZ3R/mXP5r+sKfP9cbys4RY/TZn/ikb4fweKg/YvmzL/ZbMze66v281ndv6HZucfm91rfzWmQftVl+XPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcPPmcN/8zSX9jc+c6j+q1Ndv0TMSVJT2TR/j5j7O0Ie77fsmTx22X/d7rvH+nl7fLh0/6EDjL8/IqbWUNLfTHZZ/Y+eEdM+E/0z0f/3z5Nr/7mJDkj3Gzz179/1Huv7wb1td/CO/wY=</diagram></mxfile>
```
Comment (jmckulk) [inline docs/static/drawio/repo-based-standby.xml:1]: newline :confused: :question:

**Review by jmckulk — APPROVED** [review]
LGTM

File: docs/static/drawio/repo-based-standby.xml (hunk 1 above)
Comment (tjmoore4) [reply docs/static/drawio/repo-based-standby.xml:1]: Those files are created by draw.io and it looks like that's how they are generated. I did try adding a newline and it doesn't seem to impact the file being used, but I'm not sure it's worth adding the line to an auto-generated file. What do you think about leaving it as is?

Comment (jmckulk) [reply docs/static/drawio/repo-based-standby.xml:1]: that sounds good to me :+1:

**Pull request merged**